packages/modules/auth/src/services/auth-module.ts

```diff
@@ -194,8 +196,26 @@ export default class AuthModuleService<
           authIdentities[0]
         )
       },
-      create: async (data: AuthTypes.CreateAuthIdentityDTO) => {
-        const createdAuthIdentity = await this.authIdentityService_.create(data)
+
+      create: async (data: {
```

Comment:
thought: maybe this should be renamed to `createOrLink` to make it clear what is happening?

Reply:
@srindom I already added a comment to this interface, but yes, the interface will need to change once linking support is added